cabot/cabotapp/views.py

```diff
@@ -255,25 +259,29 @@ class Meta:
                 'data-rel': 'chosen',
                 'style': 'width: 70%',
             }),
-            'users_to_notify': forms.CheckboxSelectMultiple(),
-            'schedules': forms.CheckboxSelectMultiple(),
-            'hipchat_instances': forms.SelectMultiple(attrs={
```

Comment:
`hipchat_instances` and `mattermost_instances` are removed from the `widgets` dict.  In this case, what happens?  Does it use a default widget type?

Reply:
Yup, it uses the default.